node/pegnet/winners.go

```diff
@@ -0,0 +1,82 @@
+package pegnet
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+)
+
+type MinerDominanceResult struct {
+	Start       int `json:"startheight"`
+	Stop        int `json:"stopheight"`
+	Miners      map[string]MinerDominance
+	TotalWins   int32 `json:"totalwins"`
+	TotalGraded int32 `json:"totalgraded"`
+}
+
+type MinerDominance struct {
+	Identities       []string `json:"identities"`
+	TotalWins        int32    `json:"totalwins"`
+	TotalGraded      int32    `json:"totalgraded"`
+	WinPercentage    float64  `json:"winpercent"`
+	GradedPercentage float64  `json:"gradedpercent"`
+}
+
+// SelectMinerDominance returns information around which miners are winning PEG
+// and being graded in a block range
+// Params are the start and stop block height. The stop height is inclusive.
+func (p *Pegnet) SelectMinerDominance(ctx context.Context, start, stop int) (MinerDominanceResult, error) {
+	result := MinerDominanceResult{Start: start, Stop: stop, Miners: make(map[string]MinerDominance)}
```

Comment:
If both Start and Stop are set, this is just the user input. It would be nice to update this with the real values that are queried. That way someone can submit (Start=1, Stop=250000) and get a response of something like (Start=206421, Stop=226666). This would also inform people if they didn't get the full range the requested if the node is still syncing or something

Reply:
Good idea, added that :+1: